recipe/meta.yaml

```diff
@@ -12,6 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  noarch: python
   number: 0
```

Comment:
Should we be bumping the build number?

Reply:
I don't have the permission though

Reply:
I will merge it by hand and bump it to build number 1 at the same time.

Reply:
Sounds good. Maybe we should add an issue to add this for `noarch: python`.

Reply:
@isuruf can you take care of adding the issue? I am not sure where to file it exactly.

Reply:
Sorry for the confusion @sodre. Was mentioning that as something @isuruf or myself should do. FTR it is in issue ( https://github.com/conda-forge/conda-forge-webservices/issues/125 ).